auction-app-backend/src/main/java/com/example/auctionapp/exceptions/amazon/ImageUploadException.java

```diff
@@ -0,0 +1,13 @@
+package com.example.auctionapp.exceptions.amazon;
+
+import com.example.auctionapp.exceptions.GeneralException;
+
+public class ImageUploadException extends GeneralException {
+    public ImageUploadException(String message) {
+        super(500, message);
```

Comment:
Why is this 500?

Reply:
i didnt know what other status code to use, internal server error seemed most appropriate to me to indicate that the error is on the server side